### DISCUSSION TITLE
Folder for berry scripts

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#20700] [Ideas]
Now tasmota functionality not only depends on the C++ files but also on the berry scripts, would it be an idea to create a berry folder in the tasmota repo?

I'm currently working at a berry script for Dutch Smartmeters (DSMR) for Tasmota which I publish on my own github page, but I would like to have it at some place where everyone will find it.

**Comment by sfromis** [comment]
https://tasmota.github.io/docs/Berry-Cookbook/ has examples, including  a few links to project repositories.

https://github.com/arendst/Tasmota/tree/development/tasmota/berry/drivers has some contributed Berry code.

https://github.com/tasmota/Berry_playground/tree/main/modules exists but not seeing much activity.

But the majority of projects exists independently, and I've collected a list of ones I've noticed:
http://sfromis.strangled.net/tasmota/berry/github-repositories
(not very organized or promoted)

**Comment by jeroenst** [comment]
I think https://github.com/arendst/Tasmota/tree/development/tasmota/berry is a perfect place for the berry script but I didn't find it myself.

So imo it should be documented on https://tasmota.github.io/docs/Berry-Cookbook/  and also maybe in Readme.md
